shoop/front/utils/views.py

```diff
@@ -24,3 +26,30 @@ def cache_product_things(request, products, language=None, attribute_identifiers
             language=language)
     products = cache_translations(products, (language,))
     return products
+
+
+def get_pagination_variables(request, products):
+    """
+    Used to update pagination variables to category view context
+    """
+    context = {}
+
+    limit = getattr(settings, "SHOOP_PRODUCTLIST_PRODUCT_COUNT", None)
```

Comment:
But maybe we don't like to paginate for every theme. So like this pagination is optional for CategoryView.